src/Jobs/GenerateCSVJob.php

```diff
@@ -224,10 +224,10 @@ protected function getOutputPath()
     protected function getCSVWriter()
     {
         if (!$this->writer) {
-            $csvWriter = Writer::createFromPath($this->getOutputPath(), 'w');
+            $csvWriter = Writer::createFromPath($this->getOutputPath(), 'a');
 
             $csvWriter->setDelimiter($this->Seperator);
-            $csvWriter->setNewline("\r\n"); //use windows line endings for compatibility with some csv libraries
+            $csvWriter->setNewline("");
```

Comment:
Why is the new line delimiter being changed/removed?

Reply:
Without that change there is an empty line between each record.
I guess the csv writer automatically appends on a new line in this mode, but I don't see why this is at https://www.php.net/manual/en/function.fopen.php and I don't see why in vendor/league/csv/src/Writer->addRow

Reply:
what if we completely remove this line?

Reply:
I did not expect that to work because of the \n default, but it does. I have updated the PR.